ui/src/views/Tasks/TaskIndex/ListNamespaces/index.jsx

```diff
@@ -232,7 +232,7 @@ export default class ListNamespaces extends Component {
             </Fragment>
           )}
           {!loading && !hasNamespaces && !hasIndexedTasks && isSinglePath && (
-            <Typography variant="body2">No items for this page.</Typography>
```

Comment:
Accidentally included?

Reply:
Not meant to be merged but rather just to force a netlify trigger.